lib/rules/lint-no-invalid-interactive.js

```diff
@@ -81,6 +81,11 @@ module.exports = class InvalidInteractive extends Rule {
             return;
           }
 
+          // Allow {{action "foo" on="error"}} and {{action "foo" on="load"}} on any tag
```

Comment:
Hmm, I don't think we actually support `{{action 'foo' on="load"}}` or `{{action 'foo' on="error"}}` in Ember's event dispatcher.

Here is the list of the events we listen to (this is the list of valid `{{action "foo" on="<event name here>"}}` and event handler methods on `Ember.Component`):

https://github.com/emberjs/ember.js/blob/9951eea116491f5878eef2211b5b95625261e444/packages/%40ember/-internals/views/lib/system/event_dispatcher.js#L70-L98

Reply:
Someone could have used `customEvents`. And the behaviour of the `no-invalid-interactive` rule can be configured based on tag, but not based on event name. So if load/error are in `customEvents` there would be no way to configure their ignoring.

Reply:
> Someone could have used customEvents. 

Ya, definitely true 🤔. I think using custom events is pretty obscure and uncommon, and I'd prefer to avoid adding exceptions for these things all the time when in reality they are very very rarely allowed.

Reply:
Ok, I got rid of the `{{action 'foo' on="...."}}` handling.